MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -621,9 +719,11 @@
 		AE5F8770209A082500F58FDB /* route-with-banner-instructions.json */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.json; path = "route-with-banner-instructions.json"; sourceTree = "<group>"; };
 		AE8B1B96207D2B2B003050F6 /* TunnelIntersectionManagerTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TunnelIntersectionManagerTests.swift; sourceTree = "<group>"; };
 		AE997D2121137B8B00EB0AAB /* String+LocalizedConstants.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "String+LocalizedConstants.swift"; sourceTree = "<group>"; };
+		AEA5B13321312CE1003B2961 /* CPFavoritesList.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = CPFavoritesList.swift; sourceTree = "<group>"; };
```

Comment:
This enumeration doesn’t belong in the SDK proper. It hard-codes two places as favorites, so it’s of little use to users. If we keep it around, it should be part of the example application.